### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Harrynoqs**
In this section, I added the form part where users can comment, post and like post

Objectives
Use preprocessed HTML files with embedded Ruby code.
Use layouts and templates for shared content.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:index,  :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+    resources :comments, only: [:create]
```
Comment (rustam-tolipov) [inline config/routes.rb]: - [x] I couldn't find `:new` route for comments. Kindly add `:new` route/and view the file for it. The comments controller should have new and create actions.

There should be a form in the views/comments/new.html.erb file which accepts input for creating comments.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Harrynoqs,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Harrynoqs) [reply config/routes.rb]: done

**Review by mugishasam123 — APPROVED** [review]
Hi @Harrynoqs  :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors :+1: 
- Descriptive PR :rocket: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**